<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signUp</class>
 <widget class="QDialog" name="signUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 255, 166);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="maximumSize">
      <size>
       <width>731</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="InitialPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
           </property>
           <property name="text">
            <string>Sign up as a </string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="client">
             <property name="styleSheet">
              <string notr="true">background-color:rgb(120, 255, 147);
font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
             </property>
             <property name="text">
              <string>client</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="shelter">
             <property name="styleSheet">
              <string notr="true">background-color:rgb(120, 255, 147);
font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
             </property>
             <property name="text">
              <string>Shelter Owner</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ClientSignUp">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
           </property>
           <property name="text">
            <string>General Information</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>First Name *</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Middle Name </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Last Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Gender:*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>Female</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>Male</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QRadioButton" name="radioButton_5">
             <property name="text">
              <string>Another (not listed)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_7"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Date Of Birth (mm/dd/yyyy):*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_8"/>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>email address:*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_9"/>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>phone number:*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_10"/>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Any allergies:*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_11"/>
         </item>
         <item>
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Annual Salaries:*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_12"/>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>password:*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_14"/>
         </item>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Confirm password:*</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_15"/>
         </item>
         <item>
          <widget class="QPushButton" name="Next">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(120, 255, 147);
font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ShelterSignUp">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="styleSheet">
          <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
         </property>
         <property name="text">
          <string>General Information</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Shelter name:*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_16"/>
       </item>
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Shelter address:*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_17"/>
       </item>
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Shelter hours:*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_18"/>
       </item>
       <item>
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>User Name and Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>username:*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_21"/>
       </item>
       <item>
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>password:*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_20"/>
       </item>
       <item>
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Confirm password:*</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_19"/>
       </item>
       <item>
        <widget class="QPushButton" name="Next_2">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(120, 255, 147);
font: 14pt &quot;Arial Rounded MT Bold&quot;;</string>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
